src/features/addressBook/AddressQrScanner.tsx

```diff
@@ -19,7 +19,7 @@ const AddressQrScanner = () => {
   const handleBarCodeScanned = useCallback(
     async (data: string) => {
       // scanned qr is an address string
-      if (solAddressIsValid(data)) {
+      if (solAddressIsValid(data) || isValidPublicKey(data)) {
```

Comment:
You don't need both of these right? Just the isValidPublicKey?

Reply:
`solAddressIsValid` actually checks if its onCurve, where as `isValidPublicKey` doesn't. Im good with remove it tho if we want.